packages/okta-adapter/src/definitions/deploy.ts

```diff
@@ -43,6 +100,172 @@ const createCustomizations = (): Record<string, InstanceDeployApiDefinitions> =>
+        },
+        customizations: {
+          add: [
+            {
+              request: {
+                endpoint: {
+                  path: '/api/v1/apps',
+                  method: 'post',
+                  queryArgs: {
+                    // Whether to activate the app upon creation, default is true if omitted.
+                    activate: '{activate}',
+                  },
+                },
+                context: {
+                  custom:
+                    () =>
+                    ({ change }) => ({
+                      activate: getChangeData(change).value.status === INACTIVE_STATUS ? 'false' : 'true',
+                    }),
+                },
+                transformation: {
+                  omit: [ID_FIELD, LINKS_FIELD, CUSTOM_NAME_FIELD, ...APP_POLICIES],
+                },
+              },
+              copyFromResponse: {
+                additional: {
+                  adjust: async ({ value, context }) => {
+                    const subdomain = await getSubdomainFromElementsSource(context.elementSource)
+                    if (subdomain !== undefined && isCustomApp(value as Values, subdomain)) {
+                      const createdAppName = _.get(value, NAME_FIELD)
+                      return {
+                        value: {
+                          [CUSTOM_NAME_FIELD]: createdAppName,
+                        },
+                      }
+                    }
+                    return { value: {} }
+                  },
+                },
+              },
+            },
+            ...createDeployAppPolicyRequests(),
+          ],
+          modify: [
+            {
+              request: {
+                endpoint: {
+                  path: '/api/v1/apps/{id}',
+                  method: 'put',
+                },
+                transformation: {
+                  adjust: async ({ value, context }) => {
+                    validatePlainObject(value, APPLICATION_TYPE_NAME)
+                    const transformed = getChangeData(
+                      deployment.transformRemovedValuesToNull(context.change as ModificationChange<InstanceElement>, [
+                        'settings',
+                      ]),
+                    ).value
+                    return {
+                      value: {
+                        [NAME_FIELD]: _.get(value, CUSTOM_NAME_FIELD),
+                        ..._.omit(transformed, [ID_FIELD, LINKS_FIELD, CUSTOM_NAME_FIELD, ...APP_POLICIES]),
+                      },
+                    }
+                  },
+                },
+              },
+            },
+            ...createDeployAppPolicyRequests(),
+          ],
+          remove: [
+            {
+              request: {
+                endpoint: {
+                  path: '/api/v1/apps/{applicationId}',
+                  method: 'delete',
+                },
+                context: {
+                  applicationId: '{id}',
+                },
+              },
```

Comment:
nit (also in the `activate` and `deactivate` requests)
```suggestion
              request: {
                endpoint: {
                  path: '/api/v1/apps/{id}',
                  method: 'delete',
                },
              },
```